### PR TITLE
Updated License URL for nuget packages.

### PR DESCRIPTION
**Pull request by harshjain2**
Updated License URL for nuget packages.

### CODE REVIEW

**Review by codito — APPROVED** [review]

File: src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>The cross platform Microsoft Test Platform.</description>
```
Comment (smadala) [inline src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec]: Same description for both `Microsoft.TestPlatform.Portable` and `Microsoft.TestPlatform`

File: src/package/nuspec/Microsoft.TestPlatform.nuspec [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>The cross platform Microsoft Test Platform.</description>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <licenseUrl>https://www.visualstudio.com/microsoft-visual-studio-test-platform</licenseUrl>
     <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
```
Comment (smadala) [inline src/package/nuspec/Microsoft.TestPlatform.nuspec]: Good to have tags element `<tags>vstest visual-studio unittest testplatform mstest microsoft</tags>`. Feel free to add more tags

**Comment by vinodjo** [comment]
Please add the EULA license link for the following packages Microsoft.TestPlatform & Microsoft.TestPlatform.Portable

**Comment by smadala** [comment]
@vinodjo dotnet cli uses `Microsoft.TestPlatform.CLI` nuget packages, Should we add EULA license [here](https://github.com/harshjain2/vstest/blob/d3a97e344ed3e61ce0d33452c003a51f52bd23b1/src/package/nuspec/Microsoft.TestPlatform.CLI.csproj#L5)

**Comment by harshjain2** [comment]
@smadala dotnet cli will switch to Microsoft.TestPlatform.CLI nuget package. For tracking, this issue is logged
https://github.com/Microsoft/vstest/issues/1228

**Comment by vinodjo** [comment]
Please include Microsoft.TestPlatform.CLI as well.

File: src/package/nuspec/Microsoft.TestPlatform.nuspec (hunk 2 above)
Comment (harshjain2) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: 👍

File: src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec (hunk 1 above)
Comment (harshjain2) [reply src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec]: @pvlakshm Please suggest description for these packages.

**Review by vinodjo — APPROVED** [review]
Looks good

**Comment by harshjain2** [comment]
@dotnet-bot test this please.

**Comment by harshjain2** [comment]
@dotnet-bot test Windows_NT / Release Build please.

**Comment by harshjain2** [comment]
@dotnet-bot test Windows_NT / Release Build please.

**Pull request merged**